site/content/integrate/apps/api/manifest/_index.md

```diff
@@ -0,0 +1,143 @@
+	"homepage_url": "https://github.com/mattermost/mattermost-plugin-apps/examples/go/hello",
+	"requested_permissions": [
+		"act_as_bot"
+	],
+	"requested_locations": [
+		"/channel_header",
+		"/command"
+	],
+	"http": {
+		"root_url": "http://localhost:4000"
+	}
+}
+```
+
+### Permissions
+
+These are all the permissions ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Permission)) an app can ask for:
+
+| Name                               | Description                                                                               |
+| :--------------------------------- | :---------------------------------------------------------------------------------------- |
+| `user_joined_channel_notification` | Be notified when users join channels.                                                     |
+| `act_as_bot`                       | Use Mattermost REST API as the app's bot user.                                            |
+| `act_as_user`                      | Use Mattermost REST API as connected users.                                               |
+| `act_as_admin`                     | Use Mattermost REST API as a System Admin.                                                |
+| `remote_oauth2`                    | Use remote (third-party) OAuth2 support, and will store secrets to third-party system(s). |
+| `remote_webhooks`                  | Receive webhooks from a remote (third-party) system, and process them as bot.             |
+
+
+### Locations
+
+Locations ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Location)) are spaces in the UI where the app can add interactions.
+
+| Name              | Description                                                                  |
+| :---------------- | :--------------------------------------------------------------------------- |
+| `/post_menu`      | An item in the post menu.                                                    |
+| `/channel_header` | Webapp: A button in the channel header, Mobile: An item in the channel menu. |
+| `/command`        | A slash command.                                                             |
+| `/in_post`        | Attachment embedded to a post.                                               |
+
+### Deployment Information
+
+#### `http`: Apps accessible via HTTP
+
+Provides configuration for accessing an app that is deployed exterally of Mattermost and is accessible via HTTP.
+
+| Name                               | Description                                                                               |
+| :--------------------------------- | :---------------------------------------------------------------------------------------- |
+| `root_url` | Base URL to send all calls and static asset requests to. |
+| `use_jwt`  | Include a secret-based JWT in all requests to the App. The secret must be provided by the app to the sysadmin and entered at the app's installation time. |
```

Comment:
Is not obvious that this isn't a manifest field. Moving the text out of the table would make that more clear.

Reply:
added examples for each